misk/src/main/kotlin/misk/web/interceptors/ResponseBodyMarshallerFactory.kt

```diff
@@ -15,11 +14,7 @@ class ResponseBodyMarshallerFactory @Inject internal constructor(
   @JvmSuppressWildcards private val marshallerFactories: List<Marshaller.Factory>
 ) {
   /** Returns a marshaller for [action], or null if it has no response to marshal. */
-  fun create(action: Action): Marshaller<Any>? {
-    if (action.returnType.classifier == WebSocketListener::class) {
```

Comment:
little improvement: we don’t attempt to get a marshaller for a type we’re never going to marshal